### DISCUSSION TITLE
Logic problem on locked delegations

### DISCUSSION THREAD

**Discussion by Gribnik2K** [subquery/subql#971] [Frontier TestNet - Bug Reporting]
The problem is that the indexer can lose private keys, access to the server, connection with this world, go to the astral plane, or something like that. A good and proven solution is to close the project with a 0 reward coefficient, let's say after 28 eras. Both for delegators and for an indexer. Otherwise, you can freeze your delegations for an indefinite period?
![image](https://user-images.githubusercontent.com/42679698/166821343-30cc0a76-5aab-4eab-bef2-542d9e4d9236.png)

**Comment by joesixpack** [comment]
Yeah, there should be a fail safe force claim, after like 4 weeks or so.
